src/api/include/migraphx/migraphx.hpp

```diff
@@ -1255,7 +1255,9 @@ struct experimental_custom_op_base
     virtual std::string name() const                                            = 0;
     virtual argument compute(context ctx, shape output, arguments inputs) const = 0;
     virtual shape compute_shape(shapes inputs) const                            = 0;
-    virtual ~experimental_custom_op_base()                                      = default;
+    virtual std::ptrdiff_t output_alias(shapes) const { return -1; }
+    virtual bool runs_on_offload_target() const { return true; }
```

Comment:
So we're always assuming anything not setup will always run offloaded, and not have any alias?

Reply:
Yes is defined as the default behavior.

Reply:
I think it is better if i keep `runs_on_offload_target` as a pure virtual method. made changes.

Reply:
I agree with that. I think its safe to assume that any custom op derived would need to have that defined otherwise throw an error at compile time.